centreon/doc/API/centreon-api-v22.10.yaml

```diff
@@ -3997,117 +3997,6 @@ paths:
           $ref: '#/components/responses/Forbidden'
```

Comment:
Can you update the description section as well ?

Reply:
Done